UM/Qt/Bindings/Theme.py

```diff
@@ -32,36 +35,90 @@ def __init__(self, engine, parent = None):
         self._em_height = int(QFontMetrics(QCoreApplication.instance().font()).ascent())
         self._em_width = self._em_height;
 
-        self._initializeDefaults()
+        Preferences.getInstance().addPreference("general/theme", Application.getInstance().getApplicationName())
+
+        try:
+            theme_path = Resources.getPath(Resources.Themes, Preferences.getInstance().getValue("general/theme"))
+            self.load(theme_path)
+        except FileNotFoundError:
+            pass
```

Comment:
If the theme file is not found, `self._fonts`, `self._colors` and `self._sizes` will be left uninitialised. I think it'd be good to call `_initializeDefaults` nonetheless. Also, you might want to move the win32 font workaround into `_initializeDefaults`. If the font gets reset by `load` it might not get set back properly on win32.